src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
We should always name our class to a singular expression of a business/task object.
```suggestion
public enum Color {
```